### PR TITLE
remove decimal points in area importing

### LINKED ISSUE #227: Fix Area Importing
Check if the area code has decimal or none.

### PR DESCRIPTION
**Pull request by emjay0921**

### CODE REVIEW

**Review by jeremi — CHANGES_REQUESTED** [review]
@emjay0921 please review the tests so we increase the coverage

File: spp_area/models/area_import.py [hunk 1]
```diff
@@ -212,6 +212,15 @@ def import_data(self):  # noqa: C901
...
                             # Store values to columns
+                            char_type = "Int"
+                            for char in str(admin_code):
+                                if char.isalpha():
+                                    char_type = "Char"
+                                    break
+
+                            if char_type == "Int":
+                                admin_code = int(admin_code)
+
```
Comment (nhatnm0612) [inline spp_area/models/area_import.py]:
```suggestion
                            char_type = "Int" and admin_code.isnumeric() or "Char"
                            if char_type == "Int":
                                admin_code = int(admin_code)

```

**Review by nhatnm0612 — DISMISSED** [review]

File: spp_area/models/area_import.py (hunk 1 above)
Comment (reichie020212) [reply spp_area/models/area_import.py]: @emjay0921 I second this so that we avoid the unnecessary for loop

**Review by nhatnm0612 — DISMISSED** [review]

File: spp_area/models/area_import.py (hunk 1 above)
Comment (nhatnm0612) [reply spp_area/models/area_import.py]: @emjay0921 sorry, I don't even know this column is float in your test excel file 😂, but this is something that should be improve in this code, huh ? I did not change the logic, just made it better formatted

**Comment by emjay0921** [comment]
> @emjay0921 please review the tests so we increase the coverage

All good. I raised the coverage now.

File: spp_area/tests/test_area.py [hunk 2]
```diff
@@ -1,33 +1,56 @@
...
 
         self.assertEqual(len(self.area_1.child_ids), 1)
+
+    def test_02_import_data(self):
+        import_data = self.create_import_area("test")
+        import_data.import_data()
+        import_data.save_to_area()
+
+        areas = self.env["spp.area"].search([])
+        self.assertEqual(len(areas), 21)
```
Comment (nhatnm0612) [inline spp_area/tests/test_area.py:56]:
```suggestion
        self.assertNotEqual(areas, (self.area_1 | self.area_1_child))
```
I don't think assert length equal here is a good idea

**Review by nhatnm0612 — APPROVED** [review]

**Pull request closed without merging**